tests/UnifiedSpecTests/Operation.php

```diff
@@ -166,6 +184,7 @@ private function execute()
         $object = $this->entityMap[$this->object];
         assertIsObject($object);
 
+        $this->skipIfOperationIsNotSupported($object::class);
```

Comment:
I started implementing this approach, unfortunately the object we have here is not an actual object, but a name of an object in the entity map. Since this object may not even exist at the time the operation instance is created (e.g. if it's only created later using `createEntities`), I also can't get a class name for the object being used. I'm afraid we'll have to skip when we get to `execute`.